src/content/docs/usage/dev-environments.mdx

```diff
@@ -113,10 +113,11 @@ Out of the box, Daytona can be configured to open your Dev Environment in the fo
 
 Your preferred IDE will be opened automatically and connected to your Dev Environment when using the `daytona open` command.
 
-### Forward Ports from a Workspace
```

Comment:
Seems to me like you removed the good stuff 😅

Reply:
Fixed up the headings in [39d7245](https://github.com/daytonaio/docs/pull/18/commits/39d72450a39671add849ab02da3ddaa3ba541428)